ldk/javascript/examples/self-test-loop/src/tests/whisper-update.ts

```diff
@@ -1,56 +1,29 @@
 import { whisper } from '@oliveai/ldk';
 import {
   Button,
-  Checkbox,
   ChildComponents,
-  Components,
-  Markdown,
+  Component,
+  NumberInput,
+  Select,
   TextInput,
   Whisper,
+  WhisperComponentType,
 } from '@oliveai/ldk/dist/whisper';
```

Comment:
make it `@oliveai/ldk/dist/whisper/types` please. We currently reexporting the types, but I hope we will stop at some point and it is better to reduce the future change here.